### PR TITLE
Added transformToSnakeCase utility function

### PR DESCRIPTION
**Pull request by petergeorgas**
This change brings three things, closing #6 :

1. Changes the keys of type `TestCaseRequest` from snake_case, to a more JavaScript/TypeScript friendly camelCase. 
2. Creates a `util` directory within `src`, where we can put all of our utility functions, including `transformToSnakeCase` in the `util.ts` file. 
3. Calls this function on line `113` of `keploy.ts` to make sure that we send the correct formatted keys of our object to Keploy. 

@rajatsharma how does this look?

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,3692 @@
+{
+  "name": "typescript-sdk",
```
Comment (rajatsharma) [inline package-lock.json:2]: Can you remove this file? I think this is generated by mistake and we are not using `npm` in our project.

File: src/keploy.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import HttpClient, { Request } from "./client";
+import { transformToSnakeCase } from "./util/util";
```
Comment (rajatsharma) [inline src/keploy.ts:2]: Hey can you remove this nested `util`. Just create a file `src/utils.ts`

**Review by rajatsharma — CHANGES_REQUESTED** [review]

File: src/keploy.ts (hunk 2 above)
Comment (petergeorgas) [reply src/keploy.ts]: This is now done as of 41056c1.

File: package-lock.json (hunk 1 above)
Comment (petergeorgas) [reply package-lock.json]: Oops!! Yes, that was my mistake. It is now removed.

**Pull request merged**